server/src/controllers/Events/resolver.ts

```diff
@@ -511,7 +511,7 @@ export class EventResolver {
     await updateWaitlistForUserRemoval({ event, userId: ctx.user.id });
 
     const updatedEventUser = await prisma.event_users.update({
-      data: { rsvp: { connect: { name: 'no' } } },
+      data: { rsvp: { connect: { name: 'no' } }, subscribed: false },
```

Comment:
There might be `event_reminder` that needs to be deleted. `eventUser` from line `487` needs to include `event_reminder`, and then:

```suggestion
      data: {
        rsvp: { connect: { name: 'no' } },
        subscribed: false,
        ...(eventUser.event_reminder && { event_reminder: { delete: true } }),
      },
```

Reply:
Ya, I thought It will magically stop the reminder, Thank you 👍